{
  setFailFast(false);
  parseExpression("try: 1 + 1");
  assertContainsEvent("'try' not supported, all exceptions are fatal");
  assertContainsEvent("syntax error at 'try': expected expression");
}
